core/types/types.go

```diff
@@ -94,31 +94,50 @@ type Migration struct {
 type MigrationStatus int
 
 const (
+	// NoActiveMigration indicates there is currently no migration process happening on the network.
 	NoActiveMigration MigrationStatus = iota
-	MigrationNotStarted
+
+	// ActivationPeriod represents the phase after the migration proposal has been approved by the network,
+	// but before the migration begins. During this phase, validators prepare their nodes for migration.
+	ActivationPeriod
+
+	// MigrationInProgress is the phase where the migration is actively occurring. The old and new networks
+	// run concurrently, with state changes from the old network being replicated to the new network.
 	MigrationInProgress
+
+	// MigrationCompleted indicates the migration process has successfully finished,
+	// and the old network is ready to be decommissioned.
 	MigrationCompleted
+
+	// GenesisMigration refers to the phase where the node initializes with the genesis state,
```

Comment:
This is the state where the new network is trying to sync with the old network. This will be the state till we see the "Migration completed!" message on the new chain.

Reply:
During polling mode, the node is not up yet (rpc services etc), so this command doesn't work

Reply:
I thought this was `MigrationInProgress`.

Reply:
Old nodes would be in the state: "MigrationInProgress" and move to "MigrationCompleted" state once end height is reached. Whereas New nodes start at "GenesisMigration" and then move to "NoActiveMigration" state once it syncs till the end-height block from the old chain.

Reply:
Ahhhh, oh yeah.  Thanks.  That makes sense.

Reply:
So on the doc for this, can we say that this state pertains to nodes on the new network, not the phases on the old network?